client/src/components/EditProfile/EditProfile.tsx

```diff
@@ -265,18 +281,10 @@ export default function EditProfile(): JSX.Element {
     onSubmit: (values) => handleSubmit(values),
   });
 
-  useEffect(() => {
-    const fetchProfile = async () => {
-      const data = await getProfile(loggedInUserId)();
-      setCurrentProfile(data);
-      setIsDogSitter(data.profile.isDogSitter)
-    };
-    fetchProfile();
-  }, [loggedInUserId]);
-
   return (
     <Grid className={classes.root}>
       <CssBaseline />
+      {newUser === true && <Typography className={classes.newUserHeading}>Welcome to Loving Sitter! To begin, let&apos;s start with setting up your profile. </Typography>}
```

Comment:
You can check the context to see if we have a profile and conditionally display this message. At the moment, when a user creates their profile, the message is still visible.